Configuration/EventContent/python/EventContent_cff.py

```diff
@@ -690,6 +690,10 @@ def SwapKeepAndDrop(l):
                             'keep *_hltGeneralTracks_*_*',
                             'keep *_hltInitialStepTrackSelectionHighPurity_*_*',
                             'keep *_hltHighPtTripletStepTrackSelectionHighPurity_*_*',
+                            'keep *_hltInitialStepTrackSelectionHighPuritypTTCLST_*_*',
+                            'keep *_hltInitialStepTrackSelectionHighPuritypLSTCLST_*_*',
+                            'keep *_hltInitialStepTracksT5TCLST_*_*',
+                            'keep *_hltHighPtTripletStepTrackSelectionHighPuritypLSTCLST_*_*',
```

Comment:
it's quite odd that all the changes with `hlt*` are in this file, instead of HLTrigger/Configuration/python/HLTrigger_EventContent_cff.py
It may be worth checking with Marco and Martin when the PR goes upstream

Reply:
I did what Marco did here:
https://github.com/cms-sw/cmssw/pull/46823/files
Let's see if things changed since then.